packages/react-components/react-provider/src/components/FluentProvider/useFluentProviderThemeStyleTag.ts

```diff
@@ -36,8 +36,8 @@ export const useFluentProviderThemeStyleTag = (options: Pick<FluentProviderState
   }, [theme, styleTagId]);
   const previousCssRule = usePrevious(cssRule);
 
-  if (styleTag && previousCssRule !== cssRule) {
-    const sheet = styleTag.sheet as CSSStyleSheet;
+  if (styleTag?.sheet && previousCssRule !== cssRule) {
```

Comment:
Ok, let's unblock this one. It would be great if we could at least log the error when this happens:
```ts
if (styleTag && previousCssRule !== cssRule) {
  const sheet = styleTag.sheet;

  if (sheet) {
    if (sheet.cssRules.length > 0) {
      sheet.deleteRule(0);
    }
    sheet.insertRule(cssRule, 0);
  } else if(process.env.NODE_ENV !== 'production') {
    console.error('FluentProvider: No sheet available on styleTag, styles will not be inserted into DOM.');
  }
}
```

Reply:
@miroslavstastny yup, if we go this route i'll add that logging.